caikit/runtime/service_factory.py

```diff
@@ -36,7 +38,8 @@
     TrainingInfoResponse,
 )
 from caikit.runtime import service_generation
-from caikit.runtime.service_generation.core_module_helpers import get_module_info
+
+# from caikit.runtime.service_generation.core_module_helpers import get_module_info
```

Comment:
Good to see this not used!
Can you go ahead and delete this and the other commented out unused code in the PR?

Reply:
Yes sure.